README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-In this project, we have created a simple app to decode morse code. We engineered the app using Ruby.
+In this project, we have created a class my_list and a module enumerable. The module implements a subset of the functionality of Enumerable.
 
 ## Screenshot
 
```

Comment:
- [OPTIONAL] Please the link to your screenshot is broken, kindly fix that as it will make your readme file look more professional.